chromadb/quota/simple_quota_enforcer/__init__.py

```diff
@@ -16,5 +16,9 @@ def __init__(self, system: System) -> None:
         super().__init__(system)
 
     @override
-    def enforce(self) -> None:
+    def set_context(self, context: Dict[str, Any]) -> None:
+        pass
+
+    @override
+    def enforce(self, action: str, **kwargs: Dict[str, Any]) -> None:
```

Comment:
could action here be an enum? Also feels like the kwargs should be strongly typed